src/components/DarkModeToggle.vue

```diff
@@ -0,0 +1,35 @@
+<script setup>
+import { useDark, useToggle } from '@vueuse/core';
+import VueToggle from 'vue-toggle-component';
+
+const isDark = useDark();
+const toggleDark = useToggle(isDark);
+</script>
+
+<template>
+<vue-toggle
+    :title="isDark ? 'Dark Mode' : 'Light Mode'"
+    name="Toggle Dark"
+    @click="toggleDark()"
+    class="dark-mode-toggle"
+    :class="{ dark: isDark }"
+    :toggled="isDark"
+    activeColor="#31a8ff"
```

Comment:
Is this blue only used for the toggle button itself? Or is it somehow connected to the blue in the `<style>` element in App.vue?

Reply:
This one is purely the toggle, yeah. Defaults to green otherwise 🤢